packages/contentful--app-scripts/src/utils.ts

```diff
@@ -125,13 +131,25 @@ export function getEntityFromManifest<Type extends 'actions' | 'functions'>(type
         // eslint-disable-next-line no-process-exit
         process.exit(1);
       }
+      if (hasInvalidEvent) {
+        console.log(
+          `${chalk.red(
+            'Error:',
+          )} Invalid events ${hasInvalidEvent} found in the accepts array for ${type} "${
+            item.name
+          }".`,
+        );
+        // eslint-disable-next-line no-process-exit
+        process.exit(1);
+      }
 
       // EntryFile is not used but we do want to strip it
       // eslint-disable-next-line no-unused-vars
       const { entryFile: _, ...itemWithoutEntryFile } = item;
 
       return {
         ...itemWithoutEntryFile,
+        ...(accepts !== undefined ? { accepts } : {}),
```

Comment:
```suggestion
        ...(accepts && { accepts }),
```
this will only add `{ accepts }` to the return if `accepts` exists